cluster/plan.go

```diff
@@ -610,13 +610,15 @@ func (c *Cluster) BuildKubeProxyProcess(host *hosts.Host, prefixPath string, svc
 	Binds := []string{
 		fmt.Sprintf("%s:/etc/kubernetes:z", path.Join(prefixPath, "/etc/kubernetes")),
 		"/run:/run",
+		fmt.Sprintf("%s:/lib/modules:z", path.Join(prefixPath, "/lib/modules")),
 	}
 	if host.DockerInfo.OSType == "windows" { // compatible with Windows
 		Binds = []string{
 			// put the execution binaries to the host
 			fmt.Sprintf("%s:c:/host/etc/kubernetes", path.Join(prefixPath, "/etc/kubernetes")),
 			// exchange resources with other components
 			fmt.Sprintf("%s:c:/host/run", path.Join(prefixPath, "/run")),
+			fmt.Sprintf("%s:c:/host/lib/modules", path.Join(prefixPath, "/lib/modules")),
```

Comment:
Is this needed for Windows?

Reply:
I will keep it for both linux as well as windows, since we have the volume bind-mounted for both.

Reply:
Windows agent cannot start up due to this change. @thxCode should we remove this ?

Reply:
I will be removing this change for Windows to unblock this failure.